client/src/components/Header/components/nav-links.tsx

```diff
@@ -32,8 +32,7 @@ interface NavigationLocationApi {
   apiLocation: string;
 }
 
-const { clientLocale, radioLocation, apiLocation } =
-  envData as NavigationLocationApi;
+const { clientLocale, radioLocation } = envData as NavigationLocationApi;
```

Comment:
Can you import these directly? like how you did in the model.

```ts
import { apiLocation } from '../../../../config/env.json';
```

This should clear some boilerplate and unnecessary type checks

Reply:
I sure can - but I am not too keen on touching something that is not in the scope of the PR. I like mine atomic and very, very specific.

I will do a follow-up PR if you want me to.

Reply:
no need, I am almost done with removing the eslint comment and `ts-nocheck`. I will address it there 👍